draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -2810,31 +2813,12 @@ Sun Nov  6 08:49:37 1994         ; ANSI C's asctime() format
    dynamic but the client knows the certificate that the server will
    present.) In such cases, it is important to narrow the scope of
    acceptable certificates as much as possible.
+</t>
+<t>
    In special cases, it may be appropriate for
    the client to simply ignore the server's identity, but it must be
    understood that this leaves the connection open to active attack.
 </t>
-<t>
-   If a subjectAltName extension of type dNSName is present, that &MUST;
-   be used as the identity. Otherwise, the (most specific) Common Name
-   field in the Subject field of the certificate &MUST; be used. Although
-   the use of the Common Name is existing practice, it is deprecated and
-   Certification Authorities are encouraged to use the dNSName instead.
-</t>
-<t>
-   Matching is performed using the matching rules specified by
-   <xref target="RFC5280"/>.  If more than one identity of a given type is present in
-   the certificate (e.g., more than one dNSName name, a match in any one
-   of the set is considered acceptable.) Names may contain the wildcard
-   character * which is considered to match any single domain name
-   component or component fragment. E.g., *.a.com matches foo.a.com but
-   not bar.foo.a.com. f*.com matches foo.com but not bar.com.
-</t>
-<t>
-   In some cases, the URI is specified as an IP address rather than a
-   hostname. In this case, the iPAddress subjectAltName must be present
-   in the certificate and must exactly match the IP in the URI.
-</t>
 <t>
    If the hostname does not match the identity in the certificate, user
```

Comment:
Perhaps "If the certificate is not valid for the hostname"? It is a little ambiguous whether "hostname does not match" specifically talks about the SAN list not matching or also failing to chain to a trust anchor.

Reply:
Trying to work through this, I am finding the existing description to be disjointed even after yesterday's improvements. I am going to try a shorter rewrite instead.